build/ndm-daemonset/entrypoint.sh

```diff
@@ -6,13 +6,15 @@ export GOTRACEBACK=crash
 if [ -z "$ENABLE_COREDUMP" ]; then
   ulimit -c 0
 else
+  # making sure mountpath inside the container is available
+  cd /var/openebs || (echo "OpenEBS Base directory not found" && exit)
```

Comment:
here you should use exit 1 (or any non zero value)

Reply:
Done.